src/network/RTypeClient.hpp

```diff
@@ -35,11 +36,16 @@ class RTypeClient : public UDPConnection
          */
         RTypeClient(std::string hostname, std::string port);
 
-        /**
-         * @brief This function gets all the types that will be used during program execution
-         * @brief and stores them in _compNames
-         */
-        void engineInit();
+        // /**
```

Comment:
comments left on purpose ? (line 39 to 48)

Reply:
yes